blaze/compute/tests/test_numpy_compute.py

```diff
@@ -18,6 +16,10 @@
 import blaze
 from odo import into
 from datashape import discover, to_numpy, dshape
+try:
+    import numba
+except ImportError:
+    numba = None
```

Comment:
You are correct. But it turns out we didn't actually need numba here anymore. We were checking for a specific binop (fmod) to raise the `numba.TypingError` but it isn't tested there anymore.